microsoft/devhome
# Show a dialog to the user when app execution alias is disabled and want to run PI as admin

## Summary of the pull request
When the user wants to run PI as admin and alias is disabled, show a content dialog to the user to enable alias from settings app.

## Validation steps performed
1. Launch PI
2. Disable alias for PI
3. attach to task manager
4. click on the Run as admin button
5. See the dialog
![image](https://github.com/microsoft/devhome/assets/78943958/afc5aaf5-c69e-4261-9e17-0386830cbde8)

6. Click on Go to Advanced apps settings
7. Go to alias page and enable Project Ironside
8. Click Run as admin button again on PI and confirmed PI launches as admin.

## PR checklist
- [ ] Closes #3092
